packages/manager/modules/telecom-spare/src/index.js

```diff
@@ -0,0 +1,117 @@
+import angular from 'angular';
+
+/**
+ * Here we load the "@ovh-ux/manager-core" module
+ * The body of "@ovh-ux/manager-core" executes code on the "angular" object
+ * As "angular" is a singleton, what "@ovh-ux/manager-core" executes on it
+ * is reflected here
+ */
+import '@ovh-ux/manager-core';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import ngOvhMondialRelay from '@ovh-ux/ng-ovh-mondial-relay';
+import '@ovh-ux/ng-ovh-contracts';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'oclazyload';
+import 'ovh-ui-angular';
+import 'angular-leaflet-directive';
+import 'angular-ui-bootstrap';
+import 'angular-translate-loader-partial';
+import 'angular-translate-loader-pluggable';
+import 'angular-translate-storage-cookie';
+import 'angular-translate-storage-local';
+import ngOvhSimpleCountryList from '@ovh-ux/ng-ovh-simple-country-list';
+
+// Import styles
+import 'ovh-manager-webfont/dist/css/ovh-font.css';
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css';
+import 'font-awesome/css/font-awesome.css';
+import 'ovh-common-style/dist/ovh-common-style.css';
+import 'leaflet/dist/leaflet.css';
+import 'ui-select/dist/select.css';
+import 'intl-tel-input/build/css/intlTelInput.css';
+
+import modems from './modems';
+import phones from './phones';
+import template from './template.html';
+import modemOrder from './modems/order';
+import phoneOrder from './phones/order';
+import constant from './constant';
+
+import formatModemBrandFilter from './filters/formatModemBrand/format-modem-brand.filter';
+import formatPhoneBrandFilter from './filters/formatPhoneBrand/format-phone-brand.filter';
+
+import './index.scss';
+
+const moduleName = 'ovhManagerTelecomSpare';
+
+angular
+  .module(moduleName, [
+    modems,
+    ngOvhMondialRelay,
+    ngOvhTelecomUniverseComponents,
+    modemOrder,
+    phones,
+    phoneOrder,
+    ngOvhSimpleCountryList,
+    'ngOvhContracts',
+    'oui',
+    'ngTranslateAsyncLoader',
+    'ovhManagerCore',
+    'pascalprecht.translate',
+    'ui.router',
+    'ovh-api-services',
+    'ui.bootstrap',
+    'angular-translate-loader-pluggable',
+    'ui.select',
+  ])
+  .constant('SPARE_BRAND', constant.SPARE_BRAND)
+  .config(/* @ngInject */ ($stateProvider) => {
+    $stateProvider
+      .state('spare', {
+        url: '/spare',
+        views: {
+          '': {
+            template,
+          },
+        },
+        abstract: true,
+        redirectTo: 'spare.modems',
+        resolve: {
+          me: /* @ngInject */ OvhApiMe => OvhApiMe.v6().get().$promise,
+        },
+      });
+  })
+  .filter('formatModemBrandFilter', formatModemBrandFilter)
+  .filter('formatPhoneBrandFilter', formatPhoneBrandFilter)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(/* @ngTranslationsInject:json ./common/translations */)
```

Comment:
You shouldn't have a `common` translations. each module should be able to bring its own required translations (even if it means duplicating some)